### PR TITLE
[full-ci] change: remove the settings ui

### PR DESCRIPTION
**Pull request by kulmann**
## Description
ownCloud Web recently transitioned to Vue3. The settings ui is still written in Vue2. Since it's pretty much unused we won't take the efforts of upgrading it to Vue3. The only setting ("Language") has been implemented in the `account` page of ownCloud Web (reachable via user menu in the top right -> click on the username).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by kulmann** [comment]
I had a very close look and couldn't find any more occurrences of the settings ui in config, docs, CI and of course the code itself. Please check carefully in review.

**Comment by pascalwengerter** [comment]
Guess it deserves a changelog item (but perhaps adding one was scheduled for later :) )?

**Comment by kulmann** [comment]
> Guess it deserves a changelog item (but perhaps adding one was scheduled for later :) )?

Added one, thanks for the reminder :-)

**Comment by kulmann** [comment]
Can I have a review please @micbar @tbsbdr - needs a final rebase before merging. that only makes sense after approval, so I'll wait until I have that.

**Review by dschmidt — APPROVED** [review]
Good stuff. Hard to check whether it's complete as I'm on my phone right now, but all changes look good on their own 👍️

File: changelog/unreleased/change-remove-settings-ui.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Change: Remove the settings ui
+
+With ownCloud Webs recent transition to Vue3 (which would have meant efforts for the settings ui) the decision was made
```
[the remark this answers is not in the document]
Comment (kulmann) [reply changelog/unreleased/change-remove-settings-ui.md]: 👍

**Review by micbar — APPROVED** [review]
LGTM so far. Let us move on.

**Pull request merged**